pkg/receive/handler_test.go

```diff
@@ -576,7 +579,10 @@ func testReceiveQuorum(t *testing.T, hashringAlgo HashringAlgorithm, withConsist
 		},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
-			handlers, hashring := newTestHandlerHashring(tc.appendables, tc.replicationFactor, hashringAlgo)
+			handlers, hashring, err := newTestHandlerHashring(tc.appendables, tc.replicationFactor, hashringAlgo)
+			if err != nil {
```

Comment:
We usually use `testutil.Ok`.

Reply:
I chose that to be consistent with the other assertions in this test. Its probably best to update all of them. While i'm at it should i also convert them to `testutil.Ok` form?

Reply:
I think that's a good point, let's then leave it as it is for consistency.